app/screens/backup/BackupRoot.js

```diff
@@ -0,0 +1,144 @@
+// @flow
+import { shell } from 'electron';
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import type { RouterHistory } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { WrapperWith2SideBars, Button, Link, CorneredWrapper } from '/basicComponents';
+import { fileEncryptionService } from '/infra/fileEncryptionService';
+import { fileSystemService } from '/infra/fileSystemService';
+import { localStorageService } from '/infra/storageService';
+import { smallHorizontalSideBar } from '/assets/images';
+import type { WalletMeta } from '/types';
+import smColors from '/vars/colors';
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+const SmallText = styled.span`
+  font-size: 12px;
+  font-weight: normal;
+  line-height: 20px;
+  margin-bottom: 6px;
+  flex: 1;
+`;
+
+const GreenText = styled(SmallText)`
+  color: ${smColors.green};
+`;
+
+const Text = styled.span`
+  font-size: 16px;
+  font-weight: normal;
+  line-height: 22px;
+`;
+
+const BoldText = styled(Text)`
+  font-family: SourceCodeProBold;
+  font-weight: bold;
```

Comment:
no need to add font-weight, font-family is used to bring already bold version of font